.github/workflows/python.yaml

```diff
@@ -59,3 +59,9 @@ jobs:
       run: echo "${DOCKER_PASSWORD}" | docker login -u ${DOCKER_USERNAME} --password-stdin ${DOCKER_REGISTRY_URL}
     - name: Push image
       run: docker push docker-registry.siecobywatelska.pl/small_eod:latest
+    - name: Trigger SDK regenerate
```

Comment:
To, jak rozumiem, będzie ogarnięte w #400, ale obecnie mamy u góry
```
    if: "github.ref == 'refs/heads/master'"
```
możemy to zrobić w tym pull requeście? czy ja coś gubię tutaj po drodze?

Reply:
Nic nie gubisz. Trafna uwaga.